packages/tools/src/tools/annotation/EllipticalROITool.ts

```diff
@@ -1134,16 +1134,18 @@ class EllipticalROITool extends AnnotationTool {
         pixelUnitsOptions
       );
 
-      const pointsInShape = voxelManager.forEach(
-        this.configuration.statsCalculator.statsCallback,
-        {
-          boundsIJK,
-          imageData,
-          isInObject: (pointLPS) =>
-            pointInEllipse(ellipseObj, pointLPS, { fast: true }),
-          returnPoints: this.configuration.storePointData,
-        }
-      );
+      const pointsInShape = this.configuration.storePointData
+        ? voxelManager.forEach(
+            this.configuration.statsCalculator.statsCallback,
+            {
+              boundsIJK,
+              imageData,
+              isInObject: (pointLPS) =>
+                pointInEllipse(ellipseObj, pointLPS, { fast: true }),
+              returnPoints: this.configuration.storePointData,
+            }
+          )
+        : null;
```

Comment:
I don't think this is correct. We should still perform the `statsCallback` even if `storePointData` is not `true` (calculating mean, max, std, etc.). The only purpose of `storePointData` here is to decide whether to store the points inside the elliptical or rectangular ROI in the annotation object.

Reply:
i will update the code